### ISSUE TITLE
It fails to work for Youtube

### ISSUE THREAD

**Issue by fan-wen** [aploium/zmirror-onekey#26]
Hi,

It seems it does not work for Youtube any longer. Even the demo site ytb-pc.zmirrordemo.com does not work. Could you help to update on this issue?

I appreciate your Zmirror very much, and I look forward to seeing your updates!

**Comment by seramat** [comment]
Right. Youtube video does not play, even after force to use "requests 2.11".
